packages/frontend/@n8n/i18n/src/locales/en.json

```diff
@@ -192,6 +192,8 @@
 	"auth.signup.tokenValidationError": "Issue validating invite token",
 	"aiAssistant.name": "Assistant",
 	"aiAssistant.n8nAi": "n8n AI",
+	"aiAssistant.tabs.ask": "Ask",
+	"aiAssistant.tabs.build": "Build",
 	"aiAssistant.builder.name": "Builder",
```

Comment:
Couldn't we just update the previous locale keys? I'd be fine also with adding new ones, but then let's remove the old ones(or at least `aiAssistant.builder.name` as it shouldn't be used anywhere else)

Reply:
Discussed on Slack - `aiAssistant.assistant` is still in use, but removed the `aiAssistant.builder.name` which is no longer used.